lib/Validator/LineStringValueValidator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright (C) 2016 Derek J. Lambert
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace CrEOF\Geo\Obj\Validator;
+
+use CrEOF\Geo\Obj\Configuration;
+use CrEOF\Geo\Obj\Exception\ExceptionInterface;
+use CrEOF\Geo\Obj\Exception\RangeException;
+use CrEOF\Geo\Obj\Exception\UnexpectedValueException;
+use CrEOF\Geo\Obj\ObjectInterface;
+
+/**
+ * Class LineStringValidator
+ *
+ * @author  Derek J. Lambert <dlambert@dereklambert.com>
+ * @license http://dlambert.mit-license.org MIT
+ */
+class LineStringValueValidator implements ValidatorInterface
+{
+    /**
+     * @var PointValueValidator
+     */
+    protected static $pointValidator;
+
+    /**
+     * LineStringValueValidator constructor
+     */
+    public function __construct()
+    {
+        self::$pointValidator = new PointValueValidator();
+    }
+
+    /**
+     * @param array $value
+     *
+     * @throws RangeException
+     * @throws UnexpectedValueException
+     * @throws ExceptionInterface
+     */
+    public function validate(array $value)
+    {
+        if ('linestring' !== $value['type']) {
+            return;
+        }
+
+        foreach ($value['value'] as $point) {
+            if (! is_array($point)) {
+                throw new UnexpectedValueException('LineString value must be array of "array", "' . gettype($point) . '" found');
+            }
+
+            try {
+                Configuration::getInstance()->getValidators(ObjectInterface::T_POINT)->validate([
```

Comment:
Does each point(s) in the linestring need to have the same number of coordinates ?  If so, you will probably want to add a validation here. since https://github.com/creof/geo-obj/pull/2/files#diff-f035636aaf20a157f16b58b4d6b56335R51 only verify that the number of coordinates within the point is valid